paymentsheet/src/main/java/com/stripe/android/link/LinkActivityViewModel.kt

```diff
@@ -292,6 +292,7 @@ internal class LinkActivityViewModel @Inject constructor(
                     .configuration(args.configuration)
                     .publishableKeyProvider { args.publishableKey }
                     .stripeAccountIdProvider { args.stripeAccountId }
+                    .instanceId(args.instanceId)
```

Comment:
A lot of these params aren't confined to the payment element callbacks. So a future reader might be very confused on what this actually means/does. Should we update a bunch of names to be more verbose (ie `paymentElementCallbackIdentifier`)?